src/Azure/Provider.php

```diff
@@ -63,16 +63,16 @@ public function getAccessToken($code)
     protected function getUserByToken($token)
     {
         $response = $this->getHttpClient()->get($this->graphUrl, [
-            RequestOptions::QUERY => [
-                'api-version' => $this->version,
-            ],
+            // RequestOptions::QUERY => [
```

Comment:
If this isn't required anymore, can you just remove this?

Reply:
ok done